empyrical/utils.py

```diff
@@ -205,14 +224,39 @@ def ensure_directory(path):
             raise
 
 
+def compute_returns(prices):
+    """
+    Computes correctly-indexed returns from prices.
```

Comment:
As a reader of this function, I'm not sure I would know what "correctly-indexed" means in this context. Is there something more specific we can say here?

Reply:
This used to mean 'tz-localized', but with the other comments below I don't think we want to do that after all.